src/ISM330DLCSensor.cpp

```diff
@@ -633,7 +633,7 @@ ISM330DLCStatusTypeDef ISM330DLCSensor::Set_X_ODR_When_Enabled(float odr)
 {
   ISM330DLC_ACC_GYRO_ODR_XL_t new_odr;
 
-  new_odr = ( odr <=   13.0f ) ? ISM330DLC_ACC_GYRO_ODR_XL_13Hz
+  new_odr = ( odr <=   13.0f ) ? ISM330DLC_ACC_GYRO_ODR_XL_12Hz5
```

Comment:
we need also to modify the condition in this way: "new_odr = ( odr <= 12.5f ) ? ISM330DLC_ACC_GYRO_ODR_XL_12Hz5"